receiver/hostmetricsreceiver/internal/scraper/processesscraper/internal/metadata/generated_metrics_v2.go

```diff
@@ -0,0 +1,274 @@
+// Code generated by mdatagen. DO NOT EDIT.
```

Comment:
why rename to generated_metrics_v2.go?

Reply:
We chose this filename to differentiate between the old and new generated code. `cmd/mdatagen` uses it to validate if the code is generated. One all the scrapes migrated, we will remove the suffix.

Reply:
ah, got it.